### ISSUE TITLE
Add metadata to external domain service instances with origin, path, etc

### ISSUE THREAD

**Issue by davemcorwin** [18F/federalist#3722]
#### User Story

As an operator I would like to be able to see the parameters used when creating domain services in cloud.gov. We can do this by adding metadata to the services (labels, annotations)

#### Acceptance Criteria

- [ ] appropriate metadata is added to domain services
- [ ] Change made live via deploy federalist app.

_After evaluating, edit this part:_

#### Implementation outline (if higher than "low" effort):

- [ ] <TODO>

**Comment by apburnes** [comment]
Do we want to add the update of existing services with metadata to be apart of this issue?

**Comment by apburnes** [comment]
Thoughts on use of labels vs annotations.

||Labels|Annotations
---|---|---|
|Queryable|✅ |:x:
|Value Limit| 0-63 characters| 0-5000 unicode characters

Not sure if we need the ability to query the metadata on the services but would be nice. The 63 character limit could possibly fail for sites with long organization and repository names.

**Comment by davemcorwin** [comment]
yeah, and would we want to be able to query by them? The parameters we currently set are domain names, origin, path, and optionally, custom 404 path. We're prob fine with just the annotations.
